solr/core/src/test/org/apache/solr/schema/TestPointFields.java

```diff
@@ -5289,9 +5289,9 @@ private void doTestSetQueries(String fieldName, String[] values, boolean multiVa
     SchemaField sf = h.getCore().getLatestSchema().getField(fieldName);
     assertTrue(sf.getType() instanceof PointField);
 
-    for (int i = 0; i < values.length; i++) {
+    for (String s : values) {
```

Comment:
minor: here `s` is chosen but below `value` is chosen. consistency can help with code reading e.g. `value` in both places or `s` or `v` or so.